modules/gcloud/bigquery_test.go

```diff
@@ -78,8 +80,75 @@ func ExampleRunBigQueryContainer() {
 		}
 	}
 
-	fmt.Println(val)
+	fmt.Println(val[0])
+	// Output:
+	// 30
+}
+
+func TestBigQueryWithDataYamlFile(t *testing.T) {
+	ctx := context.Background()
+
+	absPath, err := filepath.Abs(filepath.Join(".", "testdata", "data.yaml"))
+	if err != nil {
+		log.Fatalf("failed to run container: %v", err)
+	}
```

Comment:
suggestion: use require.NoError to simplify, more below
```suggestion
	require.NoError(t, err)
```